lua/autorun/client/cfc_attention_monitoring_cl_init.lua

```diff
@@ -0,0 +1,32 @@
+local tabbedOutList = {}
+local isTabbedOut = false
+local icon = Material( "icon16/monitor.png", "$translucent" )
+
+local function renderables( ply )
+    if not IsValid( ply ) then return end
+    if ply == LocalPlayer() then return end
+
+    render.SetMaterial( icon ) -- Place Holder
+    render.DrawSprite( ply:GetPos() + Vector( 0, 0, 75 ), 16, 16, Color( 225, 225, 225, 255 ) ) -- place Holder
+end
+
+timer.Create( "CFC_AttentionMonitor_tabNetTimmer", 0.5, 0, function()
+    local hasFocus = system.HasFocus()
+
+    if isTabbedOut == hasFocus then return end
+
+    net.Start( "CFC_AttentionMonitor_gameHasFocus" )
+        net.WriteBool( hasFocus )
+    net.SendToServer()
+    isTabbedOut = hasFocus
+end )
+
+net.Receive( "CFC_AttentionMonitor_sendData", function() -- receives the players that tabbed out
```

Comment:
```suggestion
net.Receive( "CFC_AttentionMonitor_SendData", function() 
```
on the server its SendData